.golangci.yaml

```diff
@@ -172,15 +172,11 @@ linters-settings:
         alias: admissioninitializer
       - pkg: github.com/gardener/gardener/pkg/chartrenderer
         alias: chartrenderer
-      - pkg: github.com/gardener/gardener/pkg/client/core/clientset/internalversion
-        alias: gardencoreclientset
       - pkg: github.com/gardener/gardener/pkg/client/core/clientset/versioned
-        alias: gardencoreversionedclientset
+        alias: gardencoreclientset
       - pkg: github.com/gardener/gardener/pkg/client/core/informers/externalversions
-        alias: gardencoreexternalinformers
-      - pkg: github.com/gardener/gardener/pkg/client/core/informers/internalversion
         alias: gardencoreinformers
-      - pkg: github.com/gardener/gardener/pkg/client/core/listers/core/internalversion
+      - pkg: github.com/gardener/gardener/pkg/client/core/listers/core/v1beta1
         alias: gardencorelisters
```

Comment:
```suggestion
        alias: gardencorev1beta1listers
```

Reply:
We have only one listers now right. I think its better to just keep it as `gardencorelisters`

Reply:
Same argument as in https://github.com/gardener/gardener/pull/8851#discussion_r1436746245